bigtable-dataflow-parent/bigtable-dataflow-import/src/test/java/com/google/protobuf/HBaseZeroCopyByteString.java

```diff
@@ -28,6 +28,28 @@
  * @since 0.96.1
  */
 public final class HBaseZeroCopyByteString  {
+
```

Comment:
How is this file used? I can't actually find any references to HBaseZeroCopyByteString anywhere...